frontend/src/components/Home.vue

```diff
@@ -58,18 +58,18 @@
       <v-col cols="12" sm="12" md="8" xs="12">
         <v-container>
           <DataTable
-            :data="EOI_MOCK"
+            :data="eoiApplications"
             title="Expressions of Interest (EOI)"
             :buttonAction="redirectToEOIForm"
             buttonTitle="Create new EOI"
-            :url="AuthRoutes.EOI"
+            routeName="expressionOfInterest"
```

Comment:
Not for now, can we bind this items to a model / context in future?

Reply:
Yes I can include that in the next PR